src/buildblock/ProjDataFromStream.cxx

```diff
@@ -250,6 +250,41 @@ ProjDataFromStream::get_bin_value(const Bin& this_bin) const
     return value[0];
 }
 
+void
+ProjDataFromStream::set_bin_value(const Bin this_bin) 
+{
+    if (is_null_ptr(sino_stream))
+    {
+        error("ProjDataFromStream::get_viewgram: stream ptr is 0\n");
```

Comment:
change text and in the ones below

Reply:
with what?

Reply:
it says `get_viewgram` ...

Reply:
oh right then should be changed to get_bin as well!